pkg/apps/controller/deployer/deployer_controller.go

```diff
@@ -176,6 +180,10 @@ func (c *DeploymentController) handle(deployment *v1.ReplicationController, will
 			} else {
 				// Update to pending or to the appropriate status relative to the existing validated deployer pod.
 				updatedAnnotations[deployapi.DeploymentPodAnnotation] = deployer.Name
+				updatedAnnotations[deployapi.DeployerPodStartedAtAnnotation] = deployer.CreationTimestamp.String()
```

Comment:
this one should be DeployerPodCreatedAtAnnotation, it was ok before

Reply:
brain fart, sorry. fixed.